CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## Changed
+
+- Updated Symfony dependencies to allow for ^4.4.
```

Comment:
@deviantintegral I think this qualifies as a minor update?

Reply:
Yep, as long as the old version still works!